seccomp.go

```diff
@@ -202,14 +202,14 @@ const (
 	// This action is only usable when libseccomp API level 3 or higher is
 	// supported.
 	ActLog
-	// ActKillThread kills the thread that violated the rule. It is the same as ActKill.
-	// All other threads from the same thread group will continue to execute.
-	ActKillThread
 	// ActKillProcess kills the process that violated the rule.
 	// All threads in the thread group are also terminated.
 	// This action is only usable when libseccomp API level 3 or higher is
 	// supported.
 	ActKillProcess
+	// ActKillThread kills the thread that violated the rule. It is the same as ActKill.
+	// All other threads from the same thread group will continue to execute.
+	ActKillThread = ActKill
```

Comment:
This may be very nitpicky, but if we are trying to be similar to the core libseccomp library it may be better to do:
```
ActKill = ActKillThread
```
... or would that still cause problems with clang?

Reply:
I tend to think it is in scope, if we are changing how we define `ActKillThread` let's go ahead and do things properly so we are better in sync with the core library.  A quick grep of the libseccomp-golang sources shows approximately two dozen places in the code that might be affected; that's not bad as far as I'm concerned.

I'm fine if you want to do this work in a separate patch, but please keep it in this PR.